7/community/run.sh

```diff
@@ -21,7 +21,7 @@ do
     fi
 done < <(env)
 
-exec tail -F ./logs/es.log &
+exec tail -F ./logs/es.log & # this tail on the elasticsearch logs is a temporary workaround, see https://github.com/docker-library/official-images/pull/6361#issuecomment-516184762
```

Comment:
I think he meant reference to our ticket

Reply:
i mean, but i found his response "better" in a way that it explain why this is not a best practice.